.github/workflows/scripts/linux/appimage-qt.sh

```diff
@@ -172,7 +172,7 @@ rm -fr "$OUTDIR/usr/bin/resources/locale"
 
 # Patch RPATH so the binary goes hunting for shared libraries in the AppDir instead of system.
 echo "Patching RPATH in ${BINARY}..."
-patchelf --set-rpath '$ORIGIN/../lib' "$OUTDIR/usr/bin/$BINARY"
+patchelf --set-rpath "$ORIGIN/../lib" "$OUTDIR/usr/bin/$BINARY"
```

Comment:
This is wrong. We **do not** want to expand the variable here, literally `$ORIGIN` is what the ELF's rpath should be set to. Same for all the others.